dlk/python/dlk/core/operators.py

```diff
@@ -612,20 +612,27 @@ def __init__(self,
                  data: np.ndarray,
                  dimension_format: str = 'NHWC',
                  packed: bool = False,
-                 actual_shape: List[int] = []) -> None:
+                 actual_shape: List[int] = [],
+                 transposed_data=None) -> None:
         """Init the variable.
 
         If the constant is hard quantized, data is packed and the actual shape
         must be expressed with `actual_shape`.
         """
         shape = list(data.shape) if not packed else actual_shape
         self._packed = packed
+        self._transposed_data = transposed_data
         super().__init__(name, shape, dtype, {}, data, dimension_format=dimension_format)
 
     @property
     def is_packed(self) -> bool:
         return self._packed
 
+    @property
+    def transposed_data(self):
```

Comment:
Please write the return `type` for consistency.

Reply:
Fixed it.